### ISSUE TITLE
`--ephemeral` flag for `environment.sh` script should be applicable also to blockchain

### ISSUE THREAD

**Issue by AuHau** [ethersphere/bee-factory#22]
We need to be able to reset the funds of the nodes. Currently `--ephemeral` flag won't persist changes in Bee's containers, but the token transfers are persisted. This I think can create inconsistencies. I think it would be good idea to propagate the `--ephemeral` flag to blockchain container as well.

**Comment by vojtechsimetka** [comment]
The caveat of course being that you need to migrate the contracts each time and fund the accounts.

**Comment by AuHau** [comment]
I don't think that is the case. Because the blockchain image is committed after all these things are done, no? So contracts deployed and funds funded, so removing container (which is what `--ephemeral` does) only resets the state to the image's state which already has all of this.

**Comment by AuHau** [comment]
Not sure if I was running wrong version, but it seems that this behavior is implemented in latest master so closing this.
